source/plugin/commands/arguments.rst

```diff
@@ -167,6 +166,12 @@ Overview of the ``GenericArguments`` command elements
 
     See the Javadocs for :javadoc:`GenericArguments` for more information.
 
+.. warning::
+
+    Some of these ``CommandElement``\s support multiple return values; some might event support regular expressions or
```

Comment:
"event" -> "even"

Also, I would say that no command element should be expected to only return one element, and mention that if you want that guarantee of only one value being returned, it must be wrapped in `GenericArguments.onlyOne(...)`.